Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -2,16 +2,73 @@
 // Licensed under the MIT License.
 
 import UIKit
+import AppCenterDistribute
 
 class MSDistributeViewController: UITableViewController, AppCenterProtocol {
 
   @IBOutlet weak var enabled: UISwitch!
   @IBOutlet weak var customized: UISwitch!
+  @IBOutlet weak var preStartSwitch: UISwitch!
+  @IBOutlet weak var updateTrackField: UITextField!
   var appCenter: AppCenterDelegate!
-  
+
+  enum UpdateTrack: String {
+     case Public = "Public"
+     case Private = "Private"
+
+     var state: MSUpdateTrack {
+        switch self {
+        case .Public: return .public
+        case .Private: return .private
+        }
+     }
+
+    static func getSelf(by track: MSUpdateTrack) -> UpdateTrack {
+       switch track {
+       case .public: return .Public
+       case .private: return .Private
+       }
+    }
+
+     static let allValues = [Public, Private]
+  }
+
+  private var updatePicker: MSEnumPicker<UpdateTrack>?
+  private var updateTrack = UpdateTrack.Public {
+    didSet {
+       self.updateTrackField.text = self.updateTrack.rawValue
+    }
+  }
+
   override func viewDidLoad() {
     super.viewDidLoad()
     self.customized.isOn = UserDefaults.init().bool(forKey: kSASCustomizedUpdateAlertKey)
+    self.preStartSwitch.isOn = UserDefaults.standard.value(forKey: kMSUpdateTrackBeforeStartValue) != nil
+
+    prepareUpdatePicker()
+
+    if let storedTrack = UserDefaults.standard.value(forKey: kMSUpdateTrackBeforeStartValue) as? Int,
+       let msTrack = MSUpdateTrack(rawValue: storedTrack) {
+        self.updateTrackField.text =  UpdateTrack.getSelf(by: msTrack).rawValue
```

Comment:
I think we should always only display the track we get from the SDK so that it always reflects the SDK state. When testing pre-start we usually remember what we put.